shell/platform/windows/flutter_platform_node_delegate_win32.cc

```diff
@@ -0,0 +1,80 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <oleacc.h>
+
+#include "flutter/shell/platform/windows/flutter_platform_node_delegate_win32.h"
+
+#include "flutter/shell/platform/windows/flutter_windows_view.h"
+
+namespace flutter {
+
+FlutterPlatformNodeDelegateWin32::FlutterPlatformNodeDelegateWin32(
+    FlutterWindowsEngine* engine)
+    : engine_(engine) {
+  assert(engine_);
+}
+
+FlutterPlatformNodeDelegateWin32::~FlutterPlatformNodeDelegateWin32() {
+  if (ax_platform_node_) {
+    ax_platform_node_->Destroy();
+  }
+}
+
+// |ui::AXPlatformNodeDelegate|
+void FlutterPlatformNodeDelegateWin32::Init(std::weak_ptr<OwnerBridge> bridge,
+                                            ui::AXNode* node) {
+  FlutterPlatformNodeDelegate::Init(bridge, node);
+  ax_platform_node_ = ui::AXPlatformNode::Create(this);
```

Comment:
Where is the fragment root created? Is it in a later patch?

Reply:
Fragment roots are only needed if we implement UIA, which is the newer a11y API. Talked to @clarkezone who mentioned we're safe with sticking with MSAA for the time being.